atlasaction/action.go

```diff
@@ -1086,6 +1086,24 @@ func appliedStmts(a *atlasexec.MigrateApply) int {
 	return total
 }
 
+func filterIssues(steps []*atlasexec.StepReport) []*atlasexec.StepReport {
+	result := make([]*atlasexec.StepReport, 0, len(steps))
+	for _, s := range steps {
+		switch {
+		case s.Error != "":
```

Comment:
looks like we can reuse `stepIsError` func
https://github.com/ariga/atlas-action/pull/283/files#diff-27a1debcaa5772fc7928822261793bc1bd5ebeca936105e889fbc6a9efc93ac5R1103

Reply:
If it's bit difference, this function also check `s.Result.Reports` but `stepIsError` didn't